### ISSUE TITLE
[4.3.0] Bug: Wrong wallet address

### ISSUE THREAD

**Issue by enjoywithouthey** [blockstack/stacks-wallet#830]
Bug found testing Stacks Wallet build undefined, 4.3.0.

Hello Stacks.

I presently hold 100 STX in wallet address: SP15X11X5ZSJXMAES64A9RVRKQ88SQ7MD8VXBYQP5 installed as a Hiro wallet on my Windows 10 Machine.

When logging into btc.us and connecting this wallet, the wallet address is read as: SP2M7K3YM8813404G1R7AXV106CPWH0Z5ZA80JVAV

How can I remedy this issue?

thanks,

-Justin

**Comment by kyranjamie** [comment]
Hi @enjoywithouthey

btc.us uses the Stacks Wallet for Web, a browser extension. This is separate from the desktop wallet, they're not intrinsically linked. It's most likely you've just created two wallets.

If you sign in to the web extension with the same seed phrase as the desktop wallet, you should have the same address.

**Comment by brunobaiano** [comment]
Hello there,
I think I will use this thread because i have i similar case.
I use the browser extension and create a account and copy the secret key.
My address was SP2G7AMJVT1ZPGAGHYBB3Y5M20T794FXAX5WRCKR4
and i could transfer some STX and try a transaction... everything works fine. this was 08/10.
Today, 08/11, I enter my secret key and my address is 
SP2E3MP2YZ1VXN2ARBW1KJPRV4J5CTZQHFZFE53RB

I really can´t see why this is happening... is there anything i can do to recover?

Has someone facing something similar?

**Comment by kyranjamie** [comment]
@brunobaiano Did you just need to select the account you previously created?

**Comment by brunobaiano** [comment]
But there is only account 1. I try create I new one now...but this old address is not on the list

**Comment by kyranjamie** [comment]
> I use the browser extension and **create a account** and copy the secret key.

Did you _Create_ a new account? Or use the _Restore_ account option? If you create a new account in the extension wallet, it doesn't reuse the seed phrase from the desktop wallet.

**Comment by brunobaiano** [comment]
I start the hiro wallet with my phrases... and it´s show only different address... Not the address i was using yesterday...
I don´t have a fortune in the wallet... but yesterday was ok... and now my secret phrases dont work anymore

>

**Comment by jeffmail** [comment]
Tonight I had the same thing happen. Has anyone ever seen or fixed this?

For the last year my stx wallet was:
SPCGY4K9AN72MBEX51DYX72R7RMV9NQ3X59MNSZQ

I have some NFTs, domain name, and 1000 STX.

Today after several months I signed back into my hiro wallet with my secret key which I have always used and store in a physical safe. It let me restore wallet by entering secret but it gave me this address:
SP1JY8YPH63GV1FDM2YBKY3F2BM75QQBCW0603PTS

I dont see my other address or tokens/etc. When I look for additional accounts it did not show any. I clicked new account and it creates new addresses, but I dont have my old one.

When I look on stx blockchain, I can find my old address and tokens/etc, but if I try and search for this new wallet address it just says no wallet found. Can anyone help?
